docs/APIs.md

```diff
@@ -0,0 +1,110 @@
+
+# APIs
+
+## API Specifications
+
+The Configuration API is specified using:
+
+* The following sub-sections describing common NMOS API properties.
+* [RAML](http://raml.org/) documents and [JSON schemas](http://tools.ietf.org/html/draft-zyp-json-schema-04) in the [APIs](../APIs/) folder.
+* Normative text in files in the [docs](../docs) directory.
+
+Examples of JSON format output are provided in the [examples](../examples/) folder.
+
+## API Validation
+
+JSON schemas are included with the RAML API definitions.
+These include validation for values used within the APIs.
+It is strongly recommended that implementers of a Configuration API use these JSON schemas as part of a validation stage when receiving interactions from clients.
```

Comment:
Is this possible since most of the schemas make references to MS-05-02 or control feature sets register datatypes?

Reply:
Hmmm, yes maybe we need to qualify this.  There are schemas in IS-14 for everything up to, but not including the MS-05 datatypes. Perhaps line 18 needs to be "...use these JSON schemas and associated schemas in MS-05-02 and the NMOS Feature Sets Register..." or something to that effect?